src/Tests/MokaMockingStrategyTestCase.php

```diff
@@ -195,11 +237,12 @@ public function testBuildWithPHP71Class()
             $this->strategy->get($this->mock)->getInt()
         );
 
-        $this->expectException(\Exception::class);
-        $this->expectExceptionMessage($this->namesWithValues['throwException']->getMessage());
```

Comment:
Why did you remove this? It's meant to test explicitly what exception is thrown, it is important to be that way.